### ISSUE TITLE
build is broken

### ISSUE THREAD

**Issue by robvanmieghem** [threefoldtech/coredns-redis#1]
```
# github.com/threefoldtech/coredns-redis
../../../../pkg/mod/github.com/threefoldtech/coredns-redis@v0.0.0-20201021094551-687617cb17d6/setup.go:24:21: cannot use c (type *"github.com/caddyserver/caddy".Controller) as type *"github.com/coredns/caddy".Controller in argument to dnsserver.GetConfig
```

**Comment by robvanmieghem** [comment]
@xmonader  fixed it by referencing the coredns caddy fork: https://github.com/xmonader/coredns-redis/commit/c13da9e076854fcc6f4eb720264984c35d1e900d

**Comment by robvanmieghem** [comment]
I do think it's strange that example plugins maintained by coredns itself like https://github.com/coredns/alternate reference `github.com/caddyserver/caddy` and use  go modules to reference `github.com/caddyserver/caddy v1.0.5`

**Comment by robvanmieghem** [comment]
@zaibon @maxux Does a change here impact the build pipeline for the gateways?

**Comment by zaibon** [comment]
It doesn't. I even think that at the moment we do not use this fork for the deployment of the TFGateway.

**Comment by robvanmieghem** [comment]
Great even though I wonder  which coredns build you do use

**Comment by xmonader** [comment]
> I do think it's strange that example plugins maintained by coredns itself like https://github.com/coredns/alternate reference `github.com/caddyserver/caddy` and use go modules to reference `github.com/caddyserver/caddy v1.0.5`

In the plugins maintained in coredns repo itself e.g forward plugin, it's using the coredns caddy fork
https://github.com/coredns/coredns/blob/master/plugin/forward/setup.go#L10

**Comment by robvanmieghem** [comment]
Thanks @xmonader for fix ad6a452769697ce3b7625b9bab4e9fcfb9432701

**Comment by zaibon** [comment]
> Great even though I wonder  which coredns build you do use

We use official coredns release. I remembered pushing upstream the fix we needed. So the fork was not needed anymore.
